ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthCreateAccessList.java

```diff
@@ -99,11 +99,15 @@ private boolean shouldProcessWithAccessListOverride(
   }
 
   private Function<TransactionSimulatorResult, Object> createResponse(
-      final JsonRpcRequestContext request, final AccessListOperationTracer operationTracer) {
+      final AccessListOperationTracer operationTracer) {
+    // access list is created regardless, but include error message if result was not successful
     return result ->
         result.isSuccessful()
             ? new CreateAccessListResult(operationTracer.getAccessList(), result.getGasEstimate())
-            : errorResponse(request, result);
+            : new CreateAccessListResult(
+                operationTracer.getAccessList(),
+                result.getGasEstimate(),
+                result.isSuccessful() ? Optional.empty() : Optional.of("execution reverted"));
```

Comment:
```suggestion
return result ->
  new new CreateAccessListResult(
                operationTracer.getAccessList(),
                result.getGasEstimate(),
                result.isSuccessful() ? Optional.empty() : Optional.of("execution reverted"));
```

and then no need for the overloaded constructor?